File: 344-reverse-string/reverse-string.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
        int l=0,r=s.size()-1;

        while(l<=r){
            char temp=s[r];
            s[r]=s[l];
            s[l]=temp;
            l++;
            r--;
        }
        for(auto c:s) cout<<c<<" ";
    }
};

File: 344-reverse-string/reverse-string.php
class Solution {

    /**
     * @param String[] $s
     * @return NULL
     */
    function reverseString(&$s) {
        $l=0;
        $r=count($s)-1;

        while($l<=$r){
            $temp=$s[$r];
            $s[$r]=$s[$l];
            $s[$l]=$temp;
            $l++;
            $r--;
        }
    }
}